.gitignore

```diff
@@ -4,6 +4,7 @@
 *.dll
 *.so
 *.dylib
+*.golangci.yml
```

Comment:
I'd rather changes to this get reviewed and committed.

Reply:
I'm not sure what that means.  Do you mean separately?  Does this not count as a review, do you mean in our meeting?

Reply:
I'd rather it not be in the .gitignore is all. If we need to change golangci.yml, then it should be changed.